operator/provider_azure_flags.go

```diff
@@ -38,5 +38,8 @@ func init() {
 	flags.String(operatorOption.AzureUserAssignedIdentityID, "", "ID of the user assigned identity used to auth with the Azure API")
 	option.BindEnvWithLegacyEnvFallback(operatorOption.AzureUserAssignedIdentityID, "AZURE_USER_ASSIGNED_IDENTITY_ID")
 
+	flags.Bool(operatorOption.AzureUsePrimaryAddress, true, "Use or ignore Azure IP address from interface's primary IPConfigurations")
```

Comment:
If I follow right from the PR description, configuring to `false` would be a regresson and therefore since this is configured to `true` by default, there should be no upgrade impact. Is that also your understanding? Just want to make sure, hence the `upgrade-impact` label on the PR.

Reply:
Exactly; there's no behaviour change (or regression, or upgrade impact) by default.
Should I re-phrase this to something more explicit, ie, like that?:

> "Use (true, default) or ignore (false) Azure IP address from interface's primary IPConfigurations

Reply:
I think just dropping the `or ignore` piece would also clarify this point.

Reply:
Thanks, right that's a simpler and clearer wording - updated.